docs/content/manual/manual.yml

```diff
@@ -568,13 +572,13 @@ sections:
 
           The | operator combines two filters by feeding the output(s) of
           the one on the left into the input of the one on the right. It's
-          pretty much the same as the Unix shell's pipe, if you're used to
-          that.
+          similar to the Unix shell's pipe, if you're used to that.
 
           If the one on the left produces multiple results, the one on
           the right will be run for each of those results. So, the
           expression `.[] | .foo` retrieves the "foo" field of each
-          element of the input array.
+          element of the input array.  This is a cartesian product,
+          which can be surprising.
```

Comment:
Maybe we need an example to clarify what this sentence means? Like `1, 2 | ., -.`. If we mention cartesian product, we may need it in the binary operator section; `(1, 2) * (3, 4)`.